src/commands/config.ts

```diff
@@ -48,8 +48,19 @@ export default class Config extends AblyBaseCommand {
     this.log(`Opening config file at ${configPath} with ${editor}...`);
 
     try {
-      // Open the editor
-      execSync(`${editor} "${configPath}"`, { stdio: "inherit" });
+      // Parse the editor string to handle cases like "open -e" or "code --wait"
+      // Split on whitespace to separate command from arguments
+      const parts = editor.split(/\s+/);
+      const command = parts[0];
+      const editorArgs = [...parts.slice(1), configPath];
+
+      // Open the editor using spawnSync with array arguments to prevent shell injection
+      // This ensures the editor and configPath are passed safely without shell interpretation
+      const result = spawnSync(command, editorArgs, { stdio: "inherit" });
```

Comment:
It looks like this is a command to open the config in an editor, which I don't think is particularly useful (I think the CLI should own the config and have semantic commands for changing things in it), so how about we avoid all this spawning stuff, have a command to print the path to the config, and allow the developer to open that in whatever way they choose if they really want to edit it?

Reply:
Nice 🙌